### PR TITLE
Support docker build multistage

### PR DESCRIPTION
**Pull request by YuviGold**

### CODE REVIEW

File: Dockerfile.test-infra [hunk 1]
```diff
@@ -1,3 +1,4 @@
+FROM quay.io/ocpmetal/bm-inventory:latest AS inventory
```
Comment (ronniel1) [inline Dockerfile.test-infra:1]: This should not always be "latest"
The tag should be passed as a m argument

Comment (tsorya) [reply Dockerfile.test-infra:1]: this should be param

Comment (tsorya) [reply Dockerfile.test-infra:1]: look here https://github.com/openshift/assisted-test-infra/pull/12/files

Comment (YuviGold) [reply Dockerfile.test-infra]: Fixed.

Comment (romfreiman) [reply Dockerfile.test-infra]: @ronniel1 @tsorya is this something that should be addressed now or can wait?

**Comment by filanov** [comment]
Worked for me.
But it wasn't easy. i had to build the image again (it wasn't in the documentation).
So please update the documentation.

**Pull request merged**